client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -47,19 +47,21 @@ export default class ProductForm extends Component {
 			}
 		);
 		return (
-			<FoldableCard
-				icon=""
-				expanded={ true }
-				className="product-variations"
-				header={ ( <FormToggle onChange={ this.handleToggle } checked={ this.state.isVariation }>
-				{variationToggleDescription}
-				</FormToggle>
-				) }
-			>
-				{ this.state.isVariation && (
-					<ProductVariationTypesForm />
-				) }
-			</FoldableCard>
+			<div className="woocommerce products__form">
```

Comment:
@coderkevin Would the best way to handle that be by wrapping these components in a simple one that returns the div and return that from the Controller?

Would you be okay with me opening an issue for that and merging this as a temporary solution for the current components and I can open another PR for that once some of the other variations work is in.